### PR TITLE
Fix GH-2655: add `PEM`, `PKCS`, `CSR` and `DER` acronyms

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/php/doc-en/issues/2655

Requires https://github.com/php/doc-base/pull/95

### CODE REVIEW

File: reference/curl/functions/curl-setopt.xml [hunk 1]
```diff
@@ -1504,11 +1504,11 @@
...
            <entry valign="top">
-             Set the pinned public key.
-             The string can be the file name of your pinned public key. The file
-             format expected is "PEM" or "DER". The string can also be any
-             number of base64 encoded sha256 hashes preceded by "sha256//" and
-             separated by ";".
+            Set the pinned public key.
+            The string can be the file name of your pinned public key. The file
+            format expected is "<acronym>PEM</acronym>" or
+            "<acronym>DER</acronym>". The string can also be any number of
```
Comment (Girgias) [inline reference/curl/functions/curl-setopt.xml:1510]: Are the quotation marks really necessary now that we have the acronym tag?

Or does this describe the string value passed as the option? In that case it should rather be wrapped by a ``<literal>`` tag

File: reference/openssl/certparams.xml [hunk 2]
```diff
@@ -21,12 +21,13 @@
...
         <filename>file://path/to/cert.pem</filename>; the named file must
-        contain a PEM encoded certificate
+        contain a <acronym>PEM</acronym> encoded certificate
        </simpara>
       </listitem>
       <listitem>
        <simpara>
-        A string containing the content of a certificate, PEM encoded, may start with -----BEGIN CERTIFICATE-----
+        A string containing the content of a certificate, <acronym>PEM</acronym>
+        encoded, may start with -----BEGIN CERTIFICATE-----
```
Comment (Girgias) [inline reference/openssl/certparams.xml:30]: Add a literal around the BEGIN CERTIFICATE while at it

File: reference/openssl/certparams.xml [hunk 3]
```diff
@@ -47,12 +48,13 @@
       <listitem>
        <simpara>A string having the format
         <filename>file://path/to/csr.pem</filename>; the named file must
-        contain a PEM encoded CSR
+        contain a <acronym>PEM</acronym> encoded CSR
```
Comment (Girgias) [inline reference/openssl/certparams.xml:51]: Does it make sense to add an acronym for CSR? As I have no idea what this means

File: reference/openssl/certparams.xml [hunk 4]
```diff
@@ -47,12 +48,13 @@
...
        </simpara>
       </listitem>
       <listitem>
        <simpara>
-        A string containing the content of a CSR, PEM encoded, may start with -----BEGIN CERTIFICATE REQUEST-----
+        A string containing the content of a CSR, <acronym>PEM</acronym> encoded,
+        may start with -----BEGIN CERTIFICATE REQUEST-----
```
Comment (Girgias) [inline reference/openssl/certparams.xml:57]: Ditto BEGIN [...] literal tag

File: reference/openssl/certparams.xml [hunk 5]
```diff
@@ -81,12 +83,14 @@
...
-        contain a PEM encoded certificate/private key (it may contain both)
+        contain a <acronym>PEM</acronym> encoded certificate/private key (it may contain both)
        </simpara>
       </listitem>
       <listitem>
        <simpara>
-        A string containing the content of a certificate/key, PEM encoded, may start with -----BEGIN PUBLIC KEY-----
+        A string containing the content of a certificate/key,
+        <acronym>PEM</acronym> encoded, may start with
+        -----BEGIN PUBLIC KEY-----
```
Comment (Girgias) [inline reference/openssl/certparams.xml:93]: Ditto BEGIN literal tag

File: reference/openssl/functions/openssl-verify.xml [hunk 6]
```diff
@@ -51,7 +51,7 @@
        <classname>OpenSSLAsymmetricKey</classname> - a key, returned by <function>openssl_get_publickey</function>
       </para>
       <para>
-       <type>string</type> - a PEM formatted key, example, "-----BEGIN PUBLIC KEY-----
+       <type>string</type> - a <acronym>PEM</acronym> formatted key, example, "-----BEGIN PUBLIC KEY-----
```
Comment (Girgias) [inline reference/openssl/functions/openssl-verify.xml:54]: Ditto literal tag

File: reference/openssl/functions/openssl-x509-verify.xml [hunk 7]
```diff
@@ -32,7 +32,7 @@
        <classname>OpenSSLAsymmetricKey</classname> - a key, returned by <function>openssl_get_publickey</function>
       </para>
       <para>
-       <type>string</type> - a PEM formatted key, example, "-----BEGIN PUBLIC KEY-----
+       <type>string</type> - a <acronym>PEM</acronym> formatted key, example, "-----BEGIN PUBLIC KEY-----
 MIIBCgK..."
```
Comment (Girgias) [inline reference/openssl/functions/openssl-x509-verify.xml:36]: Ditto literal tag

**Review by Girgias — COMMENTED** [review]
I think I would spin off the cURL set_opt() changes to another PR as that page is always a mess...

File: reference/openssl/certparams.xml (hunk 3 above)
Comment (alexandre-daubois) [reply reference/openssl/certparams.xml:51]: CRS will be added with PKCS here, I'll take care of updating files :+1: 

- https://github.com/php/doc-base/pull/96

File: reference/curl/functions/curl-setopt.xml (hunk 1 above)
Comment (alexandre-daubois) [reply reference/curl/functions/curl-setopt.xml]: I'll take care of `curl-setopt` separately :+1: I'll revert this file in this PR

**Comment by alexandre-daubois** [comment]
It should be good now @Girgias :slightly_smiling_face:

**Review by Girgias — APPROVED** [review]
I'm going to wait before merging this to let translations catch up for a bit on bug fixes. :)

**Pull request merged**